e2e/cephfs_helper.go

```diff
@@ -127,6 +127,9 @@ func createCephfsSecret(f *framework.Framework, secretName, userName, userKey st
 	if secretName != "" {
 		sc.Name = secretName
 	}
+	// TODO: Update the secrets to use userID and userKey once
+	// the version used for upgrade testing does not depend on
+	// adminID and adminKey.
```

Comment:
add a comment to remove this once we have 3.14.0 release?

Reply:
I have opened an issue for it and tracking it there. Adding the version info there.